src/scripts/components/structures/header/MainNavigation.js

```diff
@@ -4,7 +4,7 @@ import React, { Component } from 'react'
 import { Link } from 'react-router-dom'
 import styled from 'styled-components'
 
-import Button from 'components/foundations/Button'
+import Button, { ButtonColor } from 'components/foundations/Button'
```

Comment:
Can we move `ButtonColor` to another file?

Reply:
Could we do those changes with **[this](https://github.com/Paratii-Video/paratii-portal/pull/730#discussion_r196427367)** ?